react/src/pages/LoginPage.js

```diff
@@ -244,7 +244,6 @@ const LoginPage = (props) => {
                             <Tooltip id="tooltip-login-aws-alias-2" position="bottom" text={`AWS Account alias: ${header?.app?.credentials?.aws_account_name}`} />
                         </>}
                     </small>
-                        -->
```

Comment:
Weird that there was no balancing `<!--`.

Reply:
Yes, this seemed like an unintentional remnant I noticed so removed it

Reply:
Thanks